### ISSUE TITLE
Feature request: Launchpad/Launchpad mini support

### ISSUE THREAD

**Issue by Juanesgtgt2** [Aries85/reaper-csurf-launchpad-pro#1]
Hello! thanks for this plugin. have you considered including different types of launchpad to the proyect?

**Comment by Aries85** [comment]
Hi,
     thank you for the message.

TL;DR Maybe.

For details, read on.

I have been considering possibilities further development lately - from technical and funding points of view as well. There are still things to decide. One of those things is how to get devices for testing and more time for development.

I love making things work and I'd love to be able to do that continuously. There are many more ideas for features and for devices that could be made working with Reaper (all Launchpads are doable, very likely a lot of LaunchKeys as well, SL MKIII would be awesome, as well as Alesis VX49 for example). But first, more time is necessary and  some funding as well. I am confident all the things are doable.
As an example, I wrote support for another device -  Yamaha MOXF in its *DAW Remote Mode* can now control Reaper as well https://vimeo.com/339807414 And the configuration is easier than the original support for Cubase (does not require another two pieces of software installed).

Also, I am thinking about bringing the support to other software as well... but this is a distant future. 

What is needed for this project to support multiple launchpad models:
- The codebase needs to be updated as this was the my first project for REAPER and C++ language. I have done some work, but it is not release-ready.
- Adding more features - I had few ideas on what can be supported


If this can be done, many more devices could be used with Reaper conveniently. And maybe not only with Reaper... but that is distant future.
